src/main/java/org/skife/jdbi/v2/unstable/BindIn.java

```diff
@@ -25,82 +25,72 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.reflect.Method;
-import java.sql.SQLException;
 import java.util.ArrayList;
-import java.util.Collection;
 import java.util.Iterator;
 import java.util.List;
 
+/**
+ * Binds an Iterable or array/varargs to a list of arguments (e.g. for WHERE X IN (...) query statements)
+ */
 @Retention(RetentionPolicy.RUNTIME)
 @SqlStatementCustomizingAnnotation(BindIn.CustomizerFactory.class)
 @BindingAnnotation(BindIn.BindingFactory.class)
-public @interface BindIn
-{
+public @interface BindIn {
     String value();
 
-    final class CustomizerFactory implements SqlStatementCustomizerFactory
-    {
+    final class CustomizerFactory implements SqlStatementCustomizerFactory {
 
         @Override
-        public SqlStatementCustomizer createForMethod(Annotation annotation, Class sqlObjectType, Method method)
-        {
+        public SqlStatementCustomizer createForMethod(final Annotation annotation, final Class sqlObjectType, final Method method) {
             throw new UnsupportedOperationException("Not supported on method!");
         }
 
         @Override
-        public SqlStatementCustomizer createForType(Annotation annotation, Class sqlObjectType)
-        {
+        public SqlStatementCustomizer createForType(final Annotation annotation, final Class sqlObjectType) {
```

Comment:
While I share your love of K&R brace style, we're standardized on Allman brace style (open brace on the next line) on this project.

More to the point though, changing lines just for formatting makes the real history of changes opaque when you need to see the reason a line of code got to be like it is. Can you revert these lines that are just making whitespace changes?